sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionId.java

```diff
@@ -1,18 +1,15 @@
 package com.hedera.hashgraph.sdk;
 
-import com.google.protobuf.ByteString;
+
```

Comment:
Very much a nit pick, but can we keep it to one new line between `package ...` and the first `import`?